src/GitHubExtension/DeveloperId/LoginUI/LoginSucceededPage.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using GitHubExtension.Helpers;
+using Microsoft.Windows.DevHome.SDK;
+
+namespace GitHubExtension.DeveloperId.LoginUI;
+
+internal class LoginSucceededPage : LoginUIPage
+{
+    public LoginSucceededPage(IDeveloperId developerId)
+        : base(LoginUIState.LoginSucceededPage)
+    {
+        Data = new LoginSucceededPageData()
+        {
+            Message = $"{developerId.LoginId} {Resources.GetResource("LoginUI_LoginSucceededPage_text")}",
+        };
+    }
+
+    internal class LoginSucceededPageData : ILoginUIPageData
+    {
+        public string? Message { get; set; } = string.Empty;
+
+        public string GetJson()
+        {
+            return JsonSerializer.Serialize(
+            this,
+            new JsonSerializerOptions

```

Comment:
I see this serialize options being used everywhere.  Is it possible to move this seralizer options to a common file and refrence the common file?

Reply:
Ah interesting. I could maybe use a const object. Will fix.